content/source/docs/enterprise/users-teams-organizations/2fa.html.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: enterprise2
+page_title: "Two-factor Authentication - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-users-teams-organizations-2fa"
+---
+
+# Two-factor Authentication
+
+User accounts can be additionally protected with Two-factor authentication, and an organization owner can make this a requirement for all users.
+
+## Setting up Two-factor authentication
+
+To reach your user settings page, click the user icon in the upper right corner and choose "User Settings" from the menu.
+
+![The two-factor authentication page in user settings](./images/2fa-user-settings.png)
+
+Once on this page you can set-up authentication with either a TOTP-compliant Application and/or an SMS-enabled phone number.
+
+After successful set-up you will also have the ability to view backup codes, as well as disable two-factor authentication.
+
+![The two-factor authentication page after successful set-up](./images/2fa-backup-codes.png)
+
+## Logging in with Two-factor authentication
+
+After two-factor authentication has been successfully set-up you will need to enter the code from your TOTP-compliant application or from an SMS sent to your approved SMS-enabled phone number on login.
+
+If necessary you can also use a backup code by clicking "Use a recovery code". Please remember that each backup code can only be used to log in once.
```

Comment:
We call them "backup codes" in the user settings menu but on the login page we refer to them as "recovery codes". It's a minor thing but I can file it in TFE Feedback if people agree this is not an intended inconsistency.